client/dashboard/me/monetize-subscriptions/dataviews.tsx

```diff
@@ -0,0 +1,99 @@
+import { MonetizeSubscription } from '@automattic/api-core';
+import { Fields } from '@wordpress/dataviews';
+import { __ } from '@wordpress/i18n';
+import { useMemo } from 'react';
+import {
+	MonetizeSubscriptionIcon,
+	MonetizeSubscriptionTerms,
+	MonetizeSubscriptionType,
+} from './monetize-item';
+
+export function useMonetizeFieldDefinitions() {
+	return useMemo( () => {
+		return getMonetizeFieldDefinitions();
+	}, [] );
+}
+
+export function getMonetizeFieldDefinitions(): Fields< MonetizeSubscription > {
+	const getPurchaseUrl = ( item: MonetizeSubscription ) => {
+		const subscriptionId = item.ID;
+		if ( ! subscriptionId ) {
+			// eslint-disable-next-line no-console
+			console.error( 'Cannot display manage purchase page for subscription without ID' );
+			return;
+		}
+		return `/me/purchases/other/${ subscriptionId }`;
```

Comment:
Is this link to the old experience intentional?

<img width="1527" height="575" alt="Image" src="https://github.com/user-attachments/assets/b13e4101-1269-4b1e-b30c-9f9abf370f6a" />

Reply:
Fixed in 91aacc0ae276b0af23261cce7c92f6fdb1f90d15